lib/lure/command/commandinterfaces.go

```diff
@@ -20,10 +20,12 @@ type sourceControl interface {
 	Commit(string) (string, error)
 }
 
-type repository interface {
+type Repository interface {
```

Comment:
Why export this type?

Reply:
I need it in lure.go, in this part (first line):
```var provider command.Repository
switch projectConfig.Host {
case vcs.GitHub:
	provider = repository.NewGitHub(auth, projectConfig)
case vcs.Bitbucket:
	provider = repository.NewBitbucket(auth, projectConfig)
default:
	// host = nil
	err = fmt.Errorf("Unknown Host '%s' - must be one of %s, %s", projectConfig.Host, vcs.GitHub, vcs.Bitbucket)
}

Reply:
hum I see, ok lets leave it like that for now.